### DISCUSSION TITLE
Minimum Example for writing a xml

### DISCUSSION THREAD

**Discussion by Daniel63656** [pdvrieze/xmlutil#247] [General]
How can this library be used to write a simple xml (no serialization of classes, just constructing a DOM similar to pythons xml.etree)?
Can you provide a minimum working example?

**Comment by pdvrieze** [comment]
@Daniel63656 There are two ways (it only requires the core library). It also depends on your intended result. You can use various implementations of `XmlWriter` to write xml as you go (look at: [`xmlStreaming.newWriter`](https://pdvrieze.github.io/xmlutil/core/nl.adaptivity.xmlutil/-i-xml-streaming/index.html#1298882907%2FFunctions%2F459458416). Separately there are (sometimes wrapped) dom implementations you can use to manipulate the structure directly. For the dom implementation the entrypoint is domimplementation: [`xmlStreaming.genericDomImplementation`](https://pdvrieze.github.io/xmlutil/core/nl.adaptivity.xmlutil/-i-xml-streaming/index.html#-55671344%2FProperties%2F132463284)([`.createDocument`](https://pdvrieze.github.io/xmlutil/core/nl.adaptivity.xmlutil.dom2/-d-o-m-implementation/index.html#-1708153056%2FFunctions%2F132463284))
